### ISSUE TITLE
Fix geographic filter in transitland script

### ISSUE THREAD

**Issue by eric-mc2** [eric-mc2/travel-time-matrices#1]
Hey @nmarchio
Could you look at input/get_transit_land_feeds.R and help me figure out why it's filtering out every single feed? It should only be filtering out feeds that are clearly very far from the continental USA.

**Comment by nmarchio** [comment]
Hey Eric, here are some tips.

First, use this to construct a bounding box of the US:
```
us <- tigris::states(cb=TRUE) %>%
  filter(!(STUSPS %in% c('HI','AK','AS','GU','MP','PR','VI'))) %>%
  st_union() %>%
  st_transform(crs = st_crs(4326)) %>% 
  st_as_sf() 

states_bbox <- st_bbox(us)
```

And use these functions to interface with the API:
https://github.com/mountainMath/transitland/blob/master/R/transitland.R

**Comment by eric-mc2** [comment]
resolved in 6106507
